content/en/admin/optional/sso.md

```diff
@@ -12,3 +12,65 @@ This page is under construction.
 For the moment, please refer to [this pull request](https://github.com/mastodon/mastodon/pull/16221)
 {{< /hint >}}
 
+# OmniAuth
+Mastodon supports the CAS, SAML and OpenID Connect protocols for external authentication.
+These can either be configured in addition to native logins on Mastodon or as the sole
+identity provider using `OMNIAUTH_ONLY=true`.
+
+## OpenID Connect
+OpenID is configured using these environment variables.
+```
+OIDC_ENABLED=
+OIDC_DISPLAY_NAME=
+OIDC_ISSUER=
+OIDC_DISCOVERY=
+OIDC_CLIENT_AUTH_METHOD
+OIDC_SCOPE=
+OIDC_RESPONSE_TYPE=
+OIDC_RESPONSE_MODE=
+OIDC_DISPLAY=
+OIDC_PROMPT=
+OIDC_SEND_NONCE=
+OIDC_SEND_SCOPE_TO_TOKEN_ENDPOINT=
+OIDC_IDP_LOGOUT_REDIRECT_URI=
```

Comment:
This can be used in combination with `OIDC_END_SESSION_ENDPOINT` to redirect the user back to Mastodon after they have signed out.